src/components/AddAnnonce/Infos/index.jsx

```diff
@@ -0,0 +1,225 @@
+              </label>
+            </div>
+          </div>
+          <div className="boxFurnished">
+            <h2>Meublé :</h2>
+            <select value={inputFurnished} onChange={handleChangeFurnished}>
+              <option selected value="">
+                {' '}
+              </option>
+              <option value="1">Meublé</option>
+              <option value="0">Non-meublé</option>
+            </select>
+          </div>
+        </div>
+
+        <div className="box2">
+          <div className="surface">
+            <h2>Surface :</h2>
+            <input
+              type="number"
+              value={inputSurface}
+              onChange={handleChangeSurface}
+            />
+          </div>
+          <div className="rooms">
+            <h2>Nombre de pièces:</h2>
+            <div className="roomsButton">
+              <button type="button" onClick={() => seeIfSup()}>
+                -
+              </button>
+              <h2>{inputRooms}</h2>
+              <button
+                type="button"
+                onClick={() => {
+                  setRooms(inputRooms + 1);
+                }}
+              >
+                +
+              </button>
+            </div>
+          </div>
+          <div className="boxAnimals">
+            <h2>Animaux:</h2>
+            <div className="radioContainer">
+              <label htmlFor="radio_1">
+                <input
+                  id="radio_1"
+                  type="radio"
+                  name="animals"
+                  value="1"
+                  onClick={handleChangeAnimals}
+                  checked={inputAnimals === '1'}
+                />
+                Ne me dérange pas
+              </label>
+              <label htmlFor="radio_2">
+                <input
+                  id="radio_2"
+                  type="radio"
+                  name="animals"
+                  value="0"
+                  onClick={handleChangeAnimals}
+                  checked={inputAnimals === '0'}
+                />
+                Non merci
+              </label>
+            </div>
+          </div>
+        </div>
+
+        <div className="box3">
+          <div className="selectEnergy">
+            <h2>Classe énergie</h2>
+            <select value={inputEnergyClass} onChange={handleChangeEnergyClass}>
+              <option selected value="">
+                {' '}
+              </option>
+              <option value="A">A</option>
+              <option value="B">B</option>
+              <option value="C">C</option>
+              <option value="D">D</option>
+              <option value="E">E</option>
+              <option value="F">F</option>
+              <option value="G">G</option>
+            </select>
+          </div>
+
+          <div className="selectGes">
+            <h2>Gaz à effet de serre</h2>
+            <select value={inputGes} onChange={handleChangeGes}>
+              <option selected value="">
+                {' '}
```

Comment:
Same same same

Reply:
Impossible to change with the custom repo